roles/fairfood/templates/post-receive.j2

```diff
@@ -56,7 +56,7 @@ upgrade_application() {
   generate_500_page
 
   # Clean up any old gems
-  bundle clean
+  # bundle clean
```

Comment:
Why is it too eager? What did you experience?

Reply:
It shouldn't be a problem usually, but when deploying different branches on staging, it was deleting and re-downloading a lot of gems. I had to test a change for Steph yesterday, which meant re-downloading unicorn then puma again.
I thought it would be really easy to move to a cron job, there'd be less of that problem. The gems folder grows slowly so I'm thinking a weekly or even monthly task.

Reply:
My only concern with a scheduled task is that it could accidentally happen while we deploy. But we can schedule it for the middle of the night. Then it's unlikely. It would be an unsupervised change though. So if it causes a problem, we can't respond straight away. I hope that we can rely on it though. I guess that it never caused a problem so far.

The other option is to update our Ruby version more often. A new patch release is coming out several times a year...

I'm thinking too much about this. It wouldn't be a problem in production, usually. Anyway, just schedule it. One less thing during deploys.